src/app/(profile)/my-profile/(components)/ToggleOpenToWork.tsx

```diff
@@ -1,41 +1,46 @@
 'use client'
-import Switch from '@/components/Switch/Switch'
+import ImportantIcon from '@/assets/icons/ImportantIcon'
+import SwitchInput from '@/components/Switch/Switch'
+import Tooltip from '@/components/Tooltip/Tooltip'
 import { useAsyncAction } from '@/hooks/useAsyncAction'
-import { useId } from 'react'
-
+import React, { useId } from 'react'
 import { lockProfile } from '../../_actions/lockProfile'
 import { unlockProfile } from '../../_actions/unlockProfile'
+import styles from './ToogleOpenToWork.module.scss'
 
 interface ToggleOpenToWorkProps {
   profileId: string
   isOpenForWork: boolean
 }
 
-export const ToggleOpenToWork = (props: ToggleOpenToWorkProps) => {
+export const ToggleOpenToWork: React.FC<ToggleOpenToWorkProps> = ({
+  profileId,
+  isOpenForWork,
+}) => {
   const switchId = useId()
   const { runAsync } = useAsyncAction()
 
   const handleToggleOpenForWork = (isOpenForWork: boolean) => {
     runAsync(async () => {
-      const shouldLockProfile = !isOpenForWork
-
-      shouldLockProfile
-        ? await lockProfile(props.profileId)
-        : await unlockProfile(props.profileId)
+      isOpenForWork
+        ? await unlockProfile(profileId)
+        : await lockProfile(profileId)
     })
   }
 
   return (
-    <div>
-      <Switch
-        name="open-to-work"
+    <div className={styles.toggleOpenToWorkContainer}>
+      <SwitchInput
         id={switchId}
-        label={'I’m available for work'}
-        onChange={() => {
-          handleToggleOpenForWork(!props.isOpenForWork)
-        }}
-        checked={props.isOpenForWork}
-      />
+        name="open-to-work"
+        label="I’m available for work"
+        checked={isOpenForWork}
+        onChange={() => handleToggleOpenForWork(!isOpenForWork)}
+      >
+        <Tooltip text="Switch changes automatically every 7 days if you don't visit the platform. That prevents GDH from having inactive profiles.">
```

Comment:
as we don't have currently this feature, let's change it to explanation of what happens when its toggled on / off.

"Disable this feature if you prefer not to receive new contact inquiries from potential clients."